swagger_server/itm/data/metrics/scenarios/metrics-adept-eval-desert.yaml

```diff
@@ -92,7 +92,7 @@ state:
         mental_status: AGONY
         breathing: NORMAL
         heart_rate: NORMAL
-        spo2: 90
+        spo2: "LOW"
```

Comment:
For String controlled vocabulary, we've (almost?) never used double-quotes (although it works just the same).  But I suggest removing them throughout for consistency.